src/Dapr.Actors/Runtime/DaprStateProvider.cs

```diff
@@ -132,6 +133,12 @@ private async Task DoStateChangesTransactionallyAsync(string actorType, string a
                             writer.WritePropertyName("value");
                             JsonSerializer.Serialize(writer, stateChange.Value, stateChange.Type, jsonSerializerOptions);
                         }
+
+                        if (stateChange.TTLExpireTime.HasValue) {
+                            var ttl = (int)Math.Ceiling((stateChange.TTLExpireTime.Value - DateTime.UtcNow).TotalSeconds);
```

Comment:
Just a question, but is it spec'd to use the ceiling (vs. the floor or rounding) for TTLs?

Reply:
The spec is that it needs to be an integer value. It makes sense to me to be consistent in how we translate a fractional seconds. Rounding up a fractional second ensures that a value has a TTL of 1 second, rather than 0.